tools/api-markdown-documenter/src/api-item-transforms/test/Transformation.test.ts

```diff
@@ -376,11 +376,6 @@ describe("ApiItem to Documentation transformation tests", () => {
 		// The model-level doc in this case isn't particularly interesting, so we will skip evaluating it.
 
 		const expectedPackageDocument = new DocumentNode({
```

Comment:
I suspect you'll need to specify the apiItems in these tests. But you should have them already, since they're the test input.